src/applications/find-va-forms/sass/find-va-forms.scss

```diff
@@ -0,0 +1,15 @@
+.find-va-forms-table {
+  th {
+    &:nth-of-type(1) {
+      width: 120px;
+    }
+
+    &:nth-of-type(3) {
+      width: 125px;
+    }
+
+    &:nth-of-type(4) {
+      width: 145px;
+    }
+  }
+}
```

Comment:
Do we need to consider responsive behavior here at all?

Reply:
This is at 510px width:
![image](https://user-images.githubusercontent.com/12773166/71739628-35cd6480-2e17-11ea-9368-ef07b87e466c.png)

This is at 840px width:
![image](https://user-images.githubusercontent.com/12773166/71739642-45e54400-2e17-11ea-8108-2d0908732e49.png)

Reply:
We don't currently have designs for what this should look like on mobile, but I hypothesize it won't be a table as per our conversations with Liz 🙂